tools-overview.md

```diff
@@ -0,0 +1,349 @@
+[AttachmentLayout=carousel]
+
+user: thanks
+bot:
+Here's a form for you
+[Attachment=card.json adaptivecard]
+
+```
+
+### Create a transcript file from .chat file
+In the simplest form, a chatdown command looks like the following:
+```bash
+chatdown sample.chat > sample.transcript
+```
+This will consume `sample.chat` and output `sample.transcript`.
+
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Chatdown) to learn more about the Chatdown CLI.
+
+You can get the latest Bot Framework Emulator from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
+
+# Build
+
+## Bootstrap Language Understanding with Ludown
+[LUDown](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Ludown) allows you to describe and create powerful language components for bots using **.lu** files. The new .lu file is a type of markdown format which the LUDown tool consumes and outputs .json files specific to the target service. Currently, you can use .lu files to create a new [LUIS](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-get-started-create-app) application or [QnA](https://qnamaker.ai/Documentation/CreateKb) knowledge base, using different formats for each. LUDown is available as an npm module, and can be used by installing globally to your machine:
+
+```shell
+npm install -g ludown
+```
+The LUDown tool can be used to create new .json models for both LUIS and QnA.  
+
+
+### Creating a LUIS application with LUDown
+
+You can define [intents](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/add-intents) and [entities](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/add-entities) for a LUIS application just like you would from the LUIS portal. 
+
+'#\<intent-name\>' describes a new intent definition section. Each line afterwards lists the [utterances](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/add-example-utterances) which describe that intent.
+
+For example, you can create multiple LUIS intents in a single .lu file as follows: 
+
+```LUDown
+# Greeting
+- Hi
+- Hello
+- Good morning
+- Good evening
+
+# Help
+- help
+- I need help
+- please help
+```
+
+### QnA pairs with LUDown
+
+The .lu file format supports also QnA pairs using the following notation: 
+
+```LUDown
+> comment
+### ? question ?
+  ```markdown
+    answer
+  ```
+
+The LUDown tool will automatically separate question and answers into a qnamaker JSON file that you can then use to create your new [QnaMaker.ai](http://qnamaker.ai) knowledge base.
+
+```LUDown
+### ? How do I change the default message for QnA Maker?
+  ```markdown
+  You can change the default message if you use the QnAMakerDialog. 
+  See [this link](https://docs.botframework.com/en-us/azure-bot-service/templates/qnamaker/#navtitle) for details. 
+  ```
+
+You can also add multiple questions to the same answer by simply adding new lines of variations of questions for a single answer. 
```

Comment:
You can also add multiple questions **for the same answer by adding new variations** of questions for a single answer.